tools/x-repo-types/typeAnalyzer/typeAnalyzer.go

```diff
@@ -32,6 +32,9 @@ import (
 var diffExclusions = map[string]bool{
 	// MicroAlgos is a struct with custom marshal override in go-algorand. In other repos it is a uint64.
 	`github.com/algorand/go-algorand/data/basics :: "basics.MicroAlgos" (struct)`: true,
+
```

Comment:
The changes in `x-repo-types` are totally orthogonal to the CICD changes. I'm happy to break up the PR into 2 seperate ones, if that eases the review process.